test/OpenTelemetry.Tests/Metrics/AggregatorTest.cs

```diff
@@ -291,7 +291,11 @@ internal void ExponentialHistogramTests(AggregationType aggregationType, Aggrega
             foreach (var value in valuesToRecord)
             {
                 aggregatorStore.Update(value, Array.Empty<KeyValuePair<string, object>>());
-                expectedHistogram.Record(value);
+
+                if (value >= 0)
```

Comment:
I think we want to remove this `if` check in the test to make sure that negative values are not aggregated by the SDK.